src/Skia/Avalonia.Skia/Helpers/PenHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Avalonia.Media;
+
+namespace Avalonia.Skia.Helpers;
+
+internal static class PenHelper
+{
+    /// <summary>
+    /// Gets a hash code for a pen, optionally including the brush.
+    /// </summary>
+    /// <param name="pen">The pen.</param>
+    /// <param name="includeBrush">Whether to include the brush in the hash code.</param>
+    /// <returns>The hash code.</returns>
+    public static int GetHashCode(IPen? pen, bool includeBrush)
+    {
+        if (pen is null)
+            return 0;
+
+        var hash = new HashCode();
```

Comment:
I don't think it's available in netstandard2.0.

Reply:
Damn, didn't mean to push that commit yet! Yeah I was going to include a polyfil for that to Avalonia.Base. Maybe based on https://github.com/mono/SkiaSharp/blob/main/binding/Binding.Shared/HashCode.cs? Good idea or not?

Reply:
Added.